config/dev.json

```diff
@@ -7,6 +7,7 @@
   "gwasUrl": "",
   "profileUrl": "https://profile-dot-broad-shibboleth-prod.appspot.com/dev",
   "nihUrl": "https://broad-shibboleth-prod.appspot.com/dev/login",
+  "gaId": "UA-173408794-1",
```

Comment:
We will need to add real values to this field in the other config files before we push to develop. I'll get started on that today.